exercises/practice/poker/poker.spec.js

```diff
@@ -0,0 +1,176 @@
+import { bestHands } from './poker';
+
+describe('Poker', () => {
+  test('single hand always wins', () => {
+    const hands = ['4S 5S 7H 8D JC'];
+    const expected = ['4S 5S 7H 8D JC'];
+    expect(bestHands(hands)).toEqual(expected);
+  });
+
+  test('highest card out of all hands wins', () => {
```

Comment:
@SleeplessByte Should all but the first test be marked as skipped? ("xtest" instead of "test") I am not sure whether we have some strict rule about this.

Tbh I forgot to check for this in the other exercises that @lpizzinidev implemented.

Reply:
@lpizzinidev There is always lots to do, it depends on what you would like to work on. Here some possibilities:
* If you are up for creating new content from scratch, there are a lot of [concept exercises](https://exercism.org/docs/building/tracks/concept-exercises) that could be added, see https://exercism.org/docs/tracks/javascript/roadmap#h-potential-future-concepts for a list of ideas.
* Some existing concepts/ concept exercises also need improvements (see open issues).
* There are lots of outdated or missing mentor notes for practice exercises here https://github.com/exercism/website-copy/tree/main/tracks/javascript/exercises
* If you are into AST parsing, a lot of concept exercises are still missing analyzers: https://github.com/exercism/javascript-analyzer
* If you are into testing tools (Jest), the test runner needs some update: https://github.com/exercism/javascript-test-runner/issues/30
* If you are looking for something straight forward, a lot of exercises are not up to date with problem spec. They have missing test cases, the instructions changed a bit etc. The latest version of configlet can now also check and sync instructions (see https://github.com/exercism/configlet/#configlet-sync). You can get an overview of what's outdated by running `configlet sync`. I will create an issue for this soon-ish. Until then, you could just pick an exercise and submit a PR with the update.

Reply:
I think the rule is:

- concept exercise: no tests skipped
- practice exercise: all but first test skipped

Reply:
And to add to June's list! Actually making the _concept_ exercises and giving us feedback helps a lot!

Reply:
FYI I created an issue that someone can check we do the skipping correctly for all the existing practice exercises: https://github.com/exercism/javascript/issues/1666 (It's a good "first-timer" issue.)

Reply:
Thanks for the suggestions! 
I will look into the TODOs and do some more work 💪🏻